migration/gauss_hit_finder/find_hits_with_gaussians_design1.cpp

```
@@ -0,0 +1,470 @@
+            //### limits for the sum of the Hit based on the gaussian peak and sigma
+            std::vector<float>::const_iterator HitsumStartItr =
+              range.begin() + peakMean - nsigmaADC * peakWidth;
+            std::vector<float>::const_iterator HitsumEndItr =
+              range.begin() + peakMean + nsigmaADC * peakWidth;
+
+            if (nGausForFit > 1) {
+              if (numHits > 0) {
+                if ((peakMean - nsigmaADC * peakWidth) < (prevpeak + nsigmaADC * prevpeakSig)) {
+                  float difPeak = peakMean - prevpeak;
+                  float weightpeak = prevpeakSig / (prevpeakSig + peakWidth);
+                  HitsumStartItr = range.begin() + prevpeak + difPeak * weightpeak;
```

Cppcheck (2.21.0)

[style] The function 'next' is never used.

(unusedFunction)